### ISSUE TITLE
Clarify 20/20 packet distribution

### ISSUE THREAD

**Issue by carolinemao** [acf-quizbowl/acf-quizbowl.github.io#54]
Since the current distribution page on the website (https://acf-quizbowl.com/distribution/) only shows the distribution for submissions but not the actual packet

**Comment by hftf** [comment]
Isn't the final packet distribution given in the last row and explained by the asterisked footnote?

**Comment by carolinemao** [comment]
@hftf yes - Arjun said that it was unclear (which is fair as quizbowlers are not great at reading). I'm not really sure what the best solution for this is though since I'd prefer not to add a new chart
